wiki/programming/programming-interviews.md

```diff
@@ -1,8 +1,8 @@
 ---
-date: 2018-04-26
+date: {}
 title: Programming Interviews
+published: false
 ---
-
 ## Google tips
```

Comment:
This article is missing a good intro. Can you give me one paragraph? (Just make sure there's no header, links, or other markdown in the first 100 words)